pahfit/model.py

```diff
@@ -288,9 +289,12 @@ def amp_guess(row, fwhm):
 
     @staticmethod
     def _convert_spec_data(spec, z):
-        """
-        Turn astropy quantities stored in Spectrum1D into fittable
-        numbers.
+        """Convert Spectrum1D Quantities to fittable numbers.
+
+        The unit of the input spectrum has to be a multiple of MJy / sr,
+        the internal intensity unit. The output of this function
```

Comment:
We want to allow flux units at some point.  FYI, I haven't really looked much through `Model`, so don't get too wedded to any particular aspects.